src/components/UpdateProfileForm/UpdateProfileForm.jsx

```diff
@@ -83,6 +83,7 @@ function UserUpdateForm({editing, setEditing}) {
       setFormInvalid("");
       setErrorMessage("");
       setUrlError("");
+
       console.log(formData)
```

Comment:
might be a good idea to get rid of any console.logs from testing before merging to development